src/app/src/main/AndroidManifest.xml

```diff
@@ -10,7 +10,7 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.MyApplication"
+        android:theme="@style/Theme.MyApplication."
```

Comment:
what did this change?

Reply:
our style theme name had a period in it so I was just making it the same for continuity sake